src/Limbo.Integrations.Skyfish/Limbo.Integrations.Skyfish.csproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <Version>1.0.0-beta001</Version>
+    <Version>1.0.0-beta002</Version>
```

Comment:
Ideally don't change this. I prefer bumping the version number when ready to push a new release. The change is then done in a separate commit without any actual code changes - which for instance is nice if we later have to revert some code changes.